docs/getting-started.md

```diff
@@ -36,17 +36,27 @@ We designed Octopus Deploy for teams that follow agile delivery practices. A typ
 
 As an Octopus user, you define the process for deploying the software. You specify the environments the applications are deployed to and who on your team can deploy to which environments; for instance, you might want testers to deploy to test environments, but not to production. Taking this approach means that even if different members of the team trigger deployments, the deployment process remains consistent.
 
-The rest of this guide goes into more detail about working with  Octopus Deploy and links to the relevant sections of the documentation for more information.
+The rest of this guide goes into more detail about working with Octopus Deploy and links to the relevant sections of the documentation for more information.
 
-## Install Octopus Deploy
+## Octopus Deploy
 
-Installing [Octopus Deploy](/docs/installation/index.md) sets up the central [Octopus Deploy Server](/docs/installation/index.md), which provides the [Octopus REST API](/docs/api-and-integration/api/index.md) and the **Octopus Web Portal** which you will use you manage your deployments.
+Octopus Deploy is available as both a [self-hosted](/docs/getting-started.md#self-hosted-octopus) offering or as a hosted offering called [Octopus Cloud](/docs/getting-started.md#octopus-cloud).
```

Comment:
Consider rewording to: Octopus Deploy is available as both Cloud hosted and as Self-Hosted.

Reply:
When comparing like this, I've tended towards `self-hosted` and `cloud-hosted` as the descriptors (both non-capitalized), but `Octopus Cloud` when referring to the "whole platform". On it's own, `Cloud` isn't _really_ a proper noun anymore (I don't think), so it looks a bit odd to captialize it when it's abbreviated like this